src/software/backend/radio_backend.h

```diff
@@ -34,6 +36,9 @@ class RadioBackend : public Backend
     // The interface with the network that lets us get new information about the world
     NetworkClient network_input;
 
+    // Client for listener to SSL protobufs
```

Comment:
nit: Maybe "Client to listen for SSL protobufs" reads better?

Reply:
fixed